libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java

```diff
@@ -799,6 +799,14 @@ public CharSequence getContent() {
 
     @Override
     public void appendMediaFile(final MediaFile mediaFile, final String mediaUrl, ImageLoader imageLoader) {
+        if (getActivity() == null) {
```

Comment:
Niptick: should we use `isAdded()` here for consistency with other part of the code?

Reply:
Not a nitpick at all, thanks for bringing it up @daniloercoli !

Since we don't have steps to reproduce the crash, I opted for code that does exactly what's needed to guard the NPE. `isAdded()` would most probably do the job as well but, since it does some extra checks beyond making sure the Fragment's host is null or not, I'm sure if it could produce false positives.

That said, if you have better insight that `isAdded()` can be safely used in place of `getActivity()` in this case, happy to replace!